newton/tests/viewer_test_utils.py

```
@@ -0,0 +1,155 @@
+            scene: The :class:`ViewerScene` to render and advance.
+            camera: Optional ``(pos, pitch, yaw)`` overriding ``DEFAULT_CAMERA``.
+            num_frames: Optional override for ``DEFAULT_NUM_FRAMES``.
+            label: Short string used in the start-of-run log line.
+        """
+        try:
+            viewer = newton.viewer.ViewerGL()
+            viewer.set_model(scene.model)
+            cam = camera if camera is not None else self.DEFAULT_CAMERA
+            if cam is not None:
+                viewer.set_camera(pos=cam[0], pitch=cam[1], yaw=cam[2])
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)